examples/Manifest.toml

```diff
@@ -167,15 +167,15 @@ version = "0.3.3"
 
 [[deps.ClimaCoreVTK]]
 deps = ["ClimaCore", "WriteVTK"]
-path = "../lib/ClimaCoreVTK"
+git-tree-sha1 = "2a0f6f12ccdf50089e708b68e73b1fe08fb53cd8"
```

Comment:
can you point these back to the ones in the repository?

Reply:
Sure, you mean the `path` of ClimaCoreVTK, or all other changes in the Manifest? They were updated to use the latest ClimaTimeSteppers